src/app/core-ui/main/status/status.component.ts

```diff
@@ -101,7 +93,7 @@ export class StatusComponent implements OnInit, OnDestroy {
   }
 
   getStakingStatus() {
-    return (this.stakingStatus && !this.isLocked()) ? 'active' : 'inactive';
+    return (!this.isLocked()) ? 'active' : 'inactive';
```

Comment:
Put this to get data from getstakinginfo's enabled object,we dont want to show it on a wallet that has staking off

Reply:
The problem is the observer takes 5 seconds to get trigger and it will show wrong information to user, wallet not locked is already on staking enable true

Reply:
ok,id suggest just checking on init then,and not observing for the enabled status,we arent listening for staking status anymore